packages/web-twig/src/Resources/components/Accordion/AccordionHeader.twig

```diff
@@ -0,0 +1,48 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _id = props.id -%}
+{%- set _for = props.for -%}
+{%- set _elementType = (props.elementType is defined) ? props.elementType : 'h3' -%}
+{%- set _isOpen = props.isOpen -%}
```

Comment:
This option is listed as optional in the API table and thus should have a `| default()` value.

Reply:
fixed